services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/InternalRepositoryUtils.java

```diff
@@ -34,8 +35,11 @@ private InternalRepositoryUtils() {
   }
 
   public static java.nio.file.Path constructTablePath(
-      FsStorageProvider fsStorageProvider, String databaseID, String tableId, String tableUUID) {
-    return Paths.get(fsStorageProvider.rootPath(), databaseID, tableId + "-" + tableUUID);
+      StorageManager storageManager, String databaseID, String tableId, String tableUUID) {
+    return Paths.get(
+        storageManager.getDefaultStorage().getClient().getRootPath(),
```

Comment:
If a table is stored on a non-default storage, is the support for using a non-default storage coming in a future PR?

Reply:
Yes, it will come in a future PR. We'll also need to add logic to determine the non-default storage type using featureToggle.

Reply:
+1

Reply:
+1. Can you please add a TODO here indicating that we need to add support for non-default storage type per table?

Reply:
done!